src/arche/report.py

```diff
@@ -19,16 +18,16 @@ def save(self, result: Result) -> None:
         self.results[result.name] = result
 
     @staticmethod
-    def write_color_text(text: str, color: Fore = Fore.RED) -> None:
-        print(color + text + Style.RESET_ALL)
+    def write_color_text(text: str, color: str = "#8A0808") -> None:
```

Comment:
:lipstick: Maybe you could define the color here and other colors below as some constants with names similar to those of Colorama, to make the code more readable?

Reply:
I would use enum if these colors were used more than once.